### DISCUSSION TITLE
Suggestions needed for live data

### DISCUSSION THREAD

**Discussion by mmas02** [abpframework/abp#10157] [Q&A]
Hi,

I'm developing an application which required to show live odds to the users.
In the backend side, a background worker will constantly check for odds change and update into a database.
From the client side, when user visit a page, it will retrieve the latest set of records which contain all the live odds. As a result, if there are any changes on the odds, i would like to push an update the client's records so users will keep it updated without refresh the page.

Currently i'm testing the EventBus and SignalR, but i would like to get some advices from you on which approaches is much more suitable? Or is there any better solution for it?

**Comment by maliming** [comment]
I think you need **SignalR** more.
